### ISSUE TITLE
Support non-interactive mode

### ISSUE THREAD

**Issue by abatilo** [rnorth/gh-combine-prs#10]
I'd like to be able to create automated workflows that install `gh-combine-prs`
but right now that's not really possible to do naturally because the command
expects input.

**Comment by satirical-eraser** [comment]
would like to see this too for the same reasons noted

**Comment by rnorth** [comment]
Yeah, I think that'd be reasonable to add. Probably just accept a `-y` argument, and if it's present make it so that the `confirm` function continues without waiting for user input.

Would you like to raise a PR?

**Comment by abatilo** [comment]
Yep! Just opened one, thanks!

**Comment by iwahbe** [comment]
Any news on this PR?
